ganga/GangaCore/Core/GangaRepository/DatabaseRegistry.py

```diff
@@ -0,0 +1,699 @@
+
```

Comment:
This file is the same as `Registry.py`, minor changes to avoid having to acquire and release locks on jobs.